### ISSUE TITLE
Action buttons fixed on top of the card back content 

### ISSUE THREAD

**Issue by rvenugopal** [st3v3nmw/obsidian-spaced-repetition#46]
Hello

Thanks for making this plugin.  An issue I am encountering is the buttons are hovering  over the card back content instead of being fixed at the bottom.  Please see screenshot below.

Obsidian app version: 0.11.13
Obsidian spaced repetition version: 1.4.2

How I am creating the card:
The way I am making this card is using the multiline syntax (as described in the wiki) except I am using the code block as the back portion of the card.   If I knew how to look at the HTML structure I could debug this and offer a CSS fix.  However, I am not sure how to view the HTML in an electron app.

Potential solution:
One solution is to make the flashcardView as flex with column direction and overflow auto.  Then you can have a header / top section and a footer section with the buttons.  The middle div (flashcardContent ) can be sent to the markdown renderer with flex grow 1 and that would expand automatically.

```js
flashcardView.style = {
   display: flex;
   flex-directon: column;
   overflow: auto;
   flex-grow: 0 1 0;
}
```
The html can be
```html
<div id="flashcardview">
   <div></div>
   <div id="flashcardContent"></div>
   <div id="buttons"></div>
</div>
```
Current issue:
<img width="559" alt="Screen Shot 2021-05-04 at 3 15 33 PM" src="https://user-images.githubusercontent.com/234601/117077849-b04d1380-aced-11eb-8f5b-acc7352bf126.png">

**Comment by st3v3nmw** [comment]
Hello,

I'll work on fixing this.
Thanks for the bug report & the potential solution.

**Comment by rvenugopal** [comment]
👍🏽

**Comment by st3v3nmw** [comment]
I added an option to make the flashcards view scrollable in the latest release (v1.4.3). Please check it out.

**Comment by rvenugopal** [comment]
@st3v3nmw Thanks for working on this.  I just installed v1.4.4 and the modal does not even open up when I try to review

**Comment by st3v3nmw** [comment]
What Obsidian version do you have?

**Comment by st3v3nmw** [comment]
There's an API I'm using that's only available on Obsidian v0.12.2, I thought the `versions.json` file would allow older Obsidian versions to install v1.4.3.
I'll roll back to v1.4.3 for now.

**Comment by rvenugopal** [comment]
Confirming that v1.4.3  works as expected.  Thanks for resolving this.
👍🏽
